eng/docker/alpine.3.9/Dockerfile

```diff
@@ -4,7 +4,7 @@
 #
 
 # Dockerfile that creates a container suitable to build dotnet-cli
-FROM microsoft/dotnet-buildtools-prereqs:alpine-3.6-3148f11-20171119021156
+FROM microsoft/dotnet-buildtools-prereqs:alpine-3.9-helix-2e197e0-20190918214006
```

Comment:
@mthalman I find the tag on 

https://github.com/dotnet/versions/blob/master/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-master.json

But i get 

```

Step 1/13 : FROM microsoft/dotnet-buildtools-prereqs:alpine-3.9-helix-2e197e0-20190918214006
manifest for microsoft/dotnet-buildtools-prereqs:alpine-3.9-helix-2e197e0-20190918214006 not found: manifest unknown: manifest unknown
##[error]Bash exited with code '1'.
```

At the same time, I just trying out different image blindly. A different 3.9 (microsoft/dotnet-buildtools-prereqs:alpine-3.9-helix-af66924-20190215231918) image give me the following, meaning I cannot even run a Arcade restore. Is there any image in used by any Arcade supported build?

```
Target Name=InitializeSourceControlInformationFromSourceControlManager Project=Microsoft.DotNet.Tools.Tests.Utilities.csproj
    Microsoft.Build.Tasks.Git.LocateRepository
        Assembly = /home/vsts/work/1/s/.nuget/packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/../tools/netcoreapp2.0/Microsoft.Build.Tasks.Git.dll
        Parameters
            Directory = /home/vsts/work/1/s/test/Microsoft.DotNet.Tools.Tests.Utilities
        Errors
            /home/vsts/work/1/s/.nuget/packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/Microsoft.Build.Tasks.Git.targets(20,5): The type initializer for 'LibGit2Sharp.Core.NativeMethods' threw an exception.
   at LibGit2Sharp.Core.NativeMethods.git_buf_free(GitBuf buf)
   at LibGit2Sharp.Core.Proxy.git_buf_free(GitBuf buf)
   at LibGit2Sharp.Core.Handles.GitBuf.Dispose()
   at LibGit2Sharp.Core.Proxy.ConvertPath(Func`2 pathRetriever)
   at LibGit2Sharp.Core.Proxy.git_repository_discover(FilePath start_path)
   at LibGit2Sharp.Repository.Discover(String startingPath)
   at Microsoft.Build.Tasks.Git.GitOperations.LocateRepository(String directory) in /_/src/Microsoft.Build.Tasks.Git.Operations/GitOperations.cs:line 26
   at Microsoft.Build.Tasks.Git.RepositoryTasks.LocateRepository(LocateRepository task) in /_/src/Microsoft.Build.Tasks.Git.Operations/RepositoryTasks.cs:line 58 [/home/vsts/work/1/s/test/Microsoft.DotNet.Tools.Tests.Utilities/Microsoft.DotNet.Tools.Tests.Utilities.csproj]
    RepositoryType = git
    Microsoft.Build.Tasks.Git.GetRepositoryUrl
        Assembly = /home/vsts/work/1/s/.nuget/packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/../tools/netcoreapp2.0/Microsoft.Build.Tasks.Git.dll
    Microsoft.Build.Tasks.Git.GetSourceRevisionId
        Assembly = /home/vsts/work/1/s/.nuget/packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/../tools/netcoreapp2.0/Microsoft.Build.Tasks.Git.dll
    Microsoft.Build.Tasks.Git.GetSourceRoots
        Assembly = /home/vsts/work/1/s/.nuget/packages/microsoft.build.tasks.git/1.0.0-beta2-18618-05/build/../tools/netcoreapp2.0/Microsoft.Build.Tasks.Git.dll
        OutputItems
            SourceRoot = 
```

Reply:
It's because you're referencing the old repository on Docker Hub that we don't publish to any longer.  You should reference it as `mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.9-helix-2e197e0-20190918214006`.